### DISCUSSION TITLE
OException: not a regular file -- skipped

### DISCUSSION THREAD

**Discussion by Khnaz35** [Homebrew/discussions#1615] [Getting started]
Hello Team, 
I have tried to install Homebrew for my Macbook Pro 13" 2019 and when I tried to open the file i am getting this error:
```console
Last login: Sat Jun  5 20:13:39 on ttys002

The default interactive shell is now zsh.
To update your account to use zsh, please run `chsh -s /bin/zsh`.
For more details, please visit https://support.apple.com/kb/HT208050.
Mehris-MacBook-Pro:~ mehri$ sudo upx -d /Users/khnaz35/Desktop/Movavi\ PDF\ Editor\ 3.app 
Password:
                       Ultimate Packer for eXecutables
                          Copyright (C) 1996 - 2020
UPX 3.96        Markus Oberhumer, Laszlo Molnar & John Reiser   Jan 23rd 2020

        File size         Ratio      Format      Name
   --------------------   ------   -----------   -----------
upx: /Users/khnaz35/Desktop/Movavi PDF Editor 3.app: IOException: not a regular file -- skipped

Unpacked 0 files.
```
I have follow this URL to install the Homebrew: 
https://www.youtube.com/watch?v=dmNhKjHuhH0

And this was software I wanted to install:

http://oceanofdmg.com/download-movavi-pdf-editor-for-macosx/

**Comment by SMillerDev** [comment]
https://github.com/Homebrew/homebrew-cask/blob/master/doc/faq/app_cant_be_opened.md

Try reading this instead of following that YouTube tutorial. Even though none of the things you're doing are related to brew, this should resolve the issue.

**Reply by Khnaz35** [reply]
I am getting this Error (see attached ) Not the one you share that is quite easy and understandable. 
![image](https://user-images.githubusercontent.com/43812103/120892260-f7f1f280-c63f-11eb-93f4-670bec8e3eeb.png)

**Reply by SMillerDev** [reply]
Have you tried contacting the administrator as it suggests?

**Comment by Khnaz35** [comment]
I am the administrator of my system.

**Comment by nandahkrishna** [comment]
This is not a Homebrew Cask issue – there isn't a Movavi PDF Editor Cask. Maybe someone who's familiar with upx can tell you why it is unable to open the .app file.

**Reply by Khnaz35** [reply]
Thanks for reply
